pkg/util/gitutil/git.go

```diff
@@ -31,6 +31,21 @@ import (
 	"gopkg.in/src-d/go-git.v4/storage/memory"
 )
 
+// Constants related to detecting the right type of source control provider for git.
+const (
+	defaultGitCloudRepositorySuffix = ".git"
+
+	// The host name for GitLab.
+	GitLabHostName = "gitlab.com"
+	// The host name for GitHub.
+	GitHubHostName = "github.com"
+)
+
+// The pre-compiled regex used to extract owner and repo name from an SSH git remote URL.
+// CAUTION! If you are renaming any of the group names in the regex (the ?P<group_name> part) to something else,
+// be sure to update its usage in this package as well.
+var cloudSourceControlSSHRegex = regexp.MustCompile(`git@(?P<host_name>[a-zA-Z]*\.com|[a-zA-Z]*\.org):(?P<owner_and_repo>.*)`) //nolint
```

Comment:
As before, let's document why the lint warning was removed. Since in this case, I'm not sure what the problem is with the regex.